facturacion_mexico/facturacion_fiscal/tests/test_cancel_guard_draft_ffm.py

```
@@ -0,0 +1,217 @@
+			"docstatus": 1,
+			"fm_fiscal_status": "CANCELADO",
+		}
+	)
+	si.flags.ignore_validate = True
+	si.flags.ignore_mandatory = True
+	si.flags.ignore_links = True
+	si.db_insert()
+	return si.name
+
+
+def _seed_ffm(sales_invoice, status, docstatus, *, uuid=None, facturapi_id=None):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_seed_ffm`

(ANN202)